test/parallel/test-repl-history-navigation.js

```diff
@@ -554,6 +555,42 @@ const tests = [
     expected: [],
     clean: false
   },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: ['const util = {}', ENTER,
+           'ut', RIGHT, ENTER],
+    expected: [
```

Comment:
```suggestion
    expected: common.hasIntl && common.hasCrypto ? [
```

Reply:
Can I ask why does these requirements affect the console output? :)

Reply:
No clue! If you'd like to investigate that, you can compile `node` with `./configure --without-intl` on your local machine.